src/GameLogic.cpp

```diff
@@ -229,7 +230,7 @@ void GameLogic::updateProjectileCollisions() {
                     }
                     else
                     {
-                        scoreMultiplier = 1;
+                        //scoreMultiplier = 1;
```

Comment:
Have Seth playtest the game once with resetting the score multiplier back in, and see if he's able to beat it, cause I have not had a lot of trouble getting the required score to win.